### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by czubair**

### CODE REVIEW

**Review by vhvb1989 — APPROVED** [review]

**Review by vcolin7 — APPROVED** [review]

File: releases/2021-07/index.md [hunk 1]
```diff
@@ -11,9 +11,7 @@ You can find links to packages, code, and docs on our [Azure SDK Releases page](
 
 ## Release Highlights
 
-*
-*
-*
+Welcome to the July release of the Azure SDK. Read more on our [July Release Blog](https://devblogs.microsoft.com/azure-sdk/azure-sdk-release-july-2021/)
```
Comment (chidozieononiwu) [inline releases/2021-07/index.md:14]: The link on this line is a 404

**Comment by saragluna** [comment]
@czubair, the pipeline failure has been fixed in #3152. You could merge the main branch to fix it.

**Pull request merged**